app/views/layouts/_navbar.html.erb

```diff
@@ -1,5 +1,7 @@
-<nav class="navbar navbar-expand-lg navbar-light bg-light">
-  <a class="navbar-brand" href="/">WICI</a>
+<nav class="navbar navbar-expand-lg navbar-dark" style="background-color: #364f56;">
+  <a class="navbar-brand" href="/" >
```

Comment:
I think we should stick with the blue/gold already present in the logo: I'll send some color palette info ASAP. also the text doesn't contrast with the navbar enough, best to just stick with white.